include/emp/prefab/Card.hpp

```diff
@@ -9,9 +9,71 @@
 
 namespace emp {
 namespace prefab {
+
+  namespace internal {
+    /**
+     * Shared pointer held by instances of Card class representing
+     * the same conceptual Card DOM object.
+     * Contains state that should persist while Card DOM object
+     * persists.
+     */
+    class CardInfo : public web::internal::DivInfo {
+
+    public:
+      using on_toggle_fun_t = std::function<void()>;
+    protected:
+      on_toggle_fun_t toggleHandler;
+    public:
+      /**
+       * Construct a shared pointer to manage Card state.
+       * @param in_id HTML ID of Card div
+       */
+      CardInfo(const std::string & in_id="")
+      : DivInfo(in_id), toggleHandler([]() {;}) { ; }
+
+      /**
+       * Get the on-toggle function for this component
```

Comment:
Docstrings should be complete sentences ending with .’s (make a quick pass to check for this)